crates/rattler_conda_types/src/explicit_environment_spec.rs

```diff
@@ -245,6 +266,37 @@ mod test {
         );
     }
 
+    #[rstest]
+    #[case::ros_noetic_linux_64("explicit-envs/ros-noetic_linux-64.txt")]
+    #[case::vs2015_runtime_win_64("explicit-envs/vs2015_runtime_win-64.txt")]
+    #[case::xtensor_linux_64("explicit-envs/xtensor_linux-64.txt")]
+    fn test_to_path(#[case] path: &str) {
+        let env = ExplicitEnvironmentSpec::from_path(&get_test_data_dir().join(path)).unwrap();
+
+        let tmp_dir = tempfile::tempdir().unwrap();
+        let file_path = tmp_dir.path().join("temp_explicit_env.txt");
+
+        assert_matches!(env.to_path(file_path.clone()), Ok(()));
```

Comment:
```suggestion
        env.to_path(file_path.clone()).expect("failed to write to path!")
```

Reply:
I just realized I missed this one suggestion in the tests. Do you want me to open another PR?

Reply:
Its fine, dont worry about it! 👍

Thanks again for contributing!